### PR TITLE
Update location of json file.

### PR DESCRIPTION
**Pull request by tblanshard**
Update code to match new naming standard for json file, and update pom.xml to work with new location.

@andreeanica16 , @iht , @juliocc - thank you :)

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/CheckPermissionsServlet.java [hunk 1]
```diff
@@ -45,7 +45,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String projectId = request.getParameter("projID");
-    File file = new File(projectId+"-key.json");
+    File file = new File(projectId+".json");
```
Comment (juliocc) [inline src/main/java/com/google/sps/servlets/CheckPermissionsServlet.java:48]: Make sure this is not vulnerable to a Path Traversal attack (https://owasp.org/www-community/attacks/Path_Traversal). For example what happens if request.getParameter("projID") == "/etc/passwd"?

More generally, never trust user input: https://cheatsheetseries.owasp.org/cheatsheets/Input_Validation_Cheat_Sheet.html

**Review by juliocc — APPROVED** [review]
LGTM but review comments.

**Review by andreeanica16 — APPROVED** [review]
LGTM too :D

File: src/main/java/com/google/sps/servlets/CheckPermissionsServlet.java (hunk 1 above)
Comment (tblanshard) [reply src/main/java/com/google/sps/servlets/CheckPermissionsServlet.java:48]: Good point - I'll have a look at improving that section tomorrow. Thank you!

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -6,7 +6,13 @@
  */
 
 function initBody() {
-    checkPermissions();
+  setCredentialsServlet();
+  checkPermissions();
+}
+
+function setCredentialsServlet() {
+  fetch('/get-credentials?projID='+config.projectID+'&bucket='+config.bucketName+"&object="+config.objectName)
```
Comment (juliocc) [inline src/main/webapp/script.js:14]: You should encode the values using encodeURIComponent

Also, Consider using a helper function to build your URLs. I quickly put this together that you can use as a starting point:
```
function buildURL(url, params) {
    var encodedParams = new URLSearchParams(params);
    return `${url}/${encodedParams.toString()}`;
}
```
You can use it like this:
```
buildURL("my-endpoint", {param1:"foo", param2:"bar"})
-> "my-endpoint/param1=foo&param2=bar"
```

File: src/main/java/com/google/sps/servlets/SetCredentialsServlet.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                .setCredentials(GoogleCredentials.getApplicationDefault())
+                .build()
+                .getService();
+
+    Blob blob = storage.get(bucketName, objectName);
+    String fileContent = new String(blob.getContent());
+    System.out.println(fileContent);
+
+    File file = new File("pom.xml");
+    String outputPath = file.getAbsoluteFile().getParent()+"/"+projectId+".json";
```
Comment (juliocc) [inline src/main/java/com/google/sps/servlets/SetCredentialsServlet.java:62]: Careful with this. Make sure this is not prone to a path traversal attack. 

https://owasp.org/www-community/attacks/Path_Traversal

**Review by juliocc — DISMISSED** [review]
LGTM but PTAL comments.

File: src/main/webapp/script.js (hunk 2 above)
Comment (tblanshard) [reply src/main/webapp/script.js:14]: That's a lot easier and more readable - I didn't know it could be done that way! Thank you :)

**Comment by tblanshard** [comment]
Currently all the information about their projectID etc. is already in the config file, however I plan to allow user input for these (and store the user input in the config file) and that's where I intend to do the user input checks to make sure that the project isn't vulnerable to path traversal attacks etc. (which is why I haven't made the changes here). Thank you for the links (especially the generic user input one) - they're really useful!

**Review by andreeanica16 — APPROVED** [review]
LGTM

**Review by iht — APPROVED** [review]
LGTM

**Pull request merged**